CHANGES.rst

```diff
@@ -1,3 +1,11 @@
+1.8.6 (2016 August 23)
+==================
+* improve url parsing
```

Comment:
Can you elaborate on this, like _you don't need to include `api` to your host variable anymore_